dbt_subprojects/hourly_spellbook/models/_metrics/fees/metrics_fees_index_daily.sql

```diff
@@ -0,0 +1,57 @@
+{{ config(
+        schema = 'metrics'
+        , alias = 'fees_index'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+{% set baseline_date = '2018-01-01' %}
```

Comment:
what is goal of this baseline date?
only ethereum, maybe one or two other chains, will have values this far back. this will lead to nulls for other chains in calculations downstream.
do we need date per chain? if so, how do we maintain as we go?

Reply:
(note: applies to all three fees/txs/transfers spells)

Reply:
We only need the single baseline date, and it's fine for chains to not have values until they launch. That's because the index is calculating adoption overall, and each chain adds to that overall adoption as they come online. 

Here's a simplified example: on day 1 there's only ethereum. Say there's 100 transactions on that day. The next day there's 200 transactions so our index doubles. The third day a new blockchain comes online and has 100 transactions and ethereum has 200 transactions again. Now our index is 300.  The new blockchain had null values before it came online which makes sense. And it's contribution of 100 transactions can rightfully be compared to the number of transaction on the baseline day.